# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/DOMMediaStream.h
# Commit: 10d327bb5893
# Full Hash: 10d327bb5893211f030222b02456bfc11e81a943
# Author: Olli Pettay <Olli.Pettay@helsinki.fi>
# Date: 2021-05-18 21:36:28
# Regressor Bug: 1705080
# File Overlap Count: 1
# Description:
#   Bug 1705080, make MediaStreams behave as normal EventTargets, r=jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D115269
# ==============================================================================

diff -r 802480d6b25b -r 10d327bb5893 dom/media/DOMMediaStream.h
--- a/dom/media/DOMMediaStream.h	Tue May 18 16:19:44 2021 +0000
+++ b/dom/media/DOMMediaStream.h	Tue May 18 16:21:27 2021 +0000
@@ -102,7 +102,6 @@
   NS_DECL_CYCLE_COLLECTION_CLASS_INHERITED(DOMMediaStream, DOMEventTargetHelper)
   NS_DECLARE_STATIC_IID_ACCESSOR(NS_DOMMEDIASTREAM_IID)
 
-  nsPIDOMWindowInner* GetParentObject() const { return mWindow; }
   virtual JSObject* WrapObject(JSContext* aCx,
                                JS::Handle<JSObject*> aGivenProto) override;
 
@@ -224,9 +223,6 @@
   nsresult DispatchTrackEvent(const nsAString& aName,
                               const RefPtr<MediaStreamTrack>& aTrack);
 
-  // We need this to track our parent object.
-  nsCOMPtr<nsPIDOMWindowInner> mWindow;
-
   // MediaStreamTracks contained by this DOMMediaStream.
   nsTArray<RefPtr<MediaStreamTrack>> mTracks;
 